### DISCUSSION TITLE
Cant flash mod on to Asrock B450M Pro4

### DISCUSSION THREAD

**Discussion by fitsfer** [xCuri0/ReBarUEFI#175] [Q&A]
Whenever I try to flash a modded bios image to my Asrock B450M Pro4 , i get the error: "Secure Flash Check Fail!" and I'm prompted to restart. Ive tried:

- Downgrading to a [bios version](https://www.asrock.com/mb/AMD/B450M%20Pro4/index.asp#BIOS:~:text=China-,4.20,-2020/7/9) with AGESA 1.0.0.6 beforehand
- using [UBU tool](https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627#:~:text=BIOS%20properly%20flashed.-,Solution%3A,-The%20%E2%80%9CSecurity%20Flash) to try to remove the security check (the filesize doesn't change)
- different file extensions

any help would be appreciated!

**Comment by MrAnonymous7122** [comment]
I would assume it's still failing the security check, OEM update software usually doesn't allow modded bios at all so you would have to try a different method for flashing.

**Comment by DeLuX-9** [comment]
> Whenever I try to flash a modded bios image to my Asrock B450M Pro4 , i get the error: "Secure Flash Check Fail!" and I'm prompted to restart. Ive tried:
> 
> * Downgrading to a [bios version](https://www.asrock.com/mb/AMD/B450M%20Pro4/index.asp#BIOS:~:text=China-,4.20,-2020/7/9) with AGESA 1.0.0.6 beforehand
> * using [UBU tool](https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627#:~:text=BIOS%20properly%20flashed.-,Solution%3A,-The%20%E2%80%9CSecurity%20Flash) to try to remove the security check (the filesize doesn't change)
> * different file extensions
> 
> any help would be appreciated!

Hi @fitsfer 

I know this is near enough a year delay... I was wondering if you had success for this at all?

**Reply by ykcgw** [reply]
Yes, me too, because I have similar board AsRock B450 Pro4
